app/workers/repository_maintenance_stat_worker.rb

```diff
@@ -1,25 +1,8 @@
 class RepositoryMaintenanceStatWorker
     include Sidekiq::Worker
-    sidekiq_options queue: :repo_maintenance_stat, retry: 3
+    sidekiq_options queue: :low, retry: 3
 
     def perform(repo_id)
         Repository.find(repo_id).gather_maintenance_stats
     end
-
-    def self.enqueue(repo_id, priority: :medium)
```

Comment:
These priorities were to let us intentionally get the first set of maintenance stats "fast" and refreshes to be able to be lower priority

Reply:
I'm not seeing very many of these queued relative to the others, can I just make them all critical?

Reply:
Making them all critical makes it so we can overload the system in a way that we really don't want to allow. Especially with the changed processing order.